### PR TITLE
blank structure global config messages

### PR DESCRIPTION
**Pull request by ritugohel1750**

### CODE REVIEW

**Review by GohilAmit — APPROVED** [review]

**Review by jayk-patel-tatvasoft — APPROVED** [review]

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        {
+            if (ModelState.IsValid)
+            {
+                try
+                {
+                    return await globalConfigurationMessagesService.CreateOrUpdateGlobalConfigurationMessages(globalConfigurationMessages);
+                }
+                catch (Exception ex)
+                {
+                    return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs:62]: need to change catch return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                }
+            }
+            return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.BadRequest, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.BadRequest.GetDescriptionAttribute(), Data = SystemMessages.msgBadRequest });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs:65]: need to changereturn response message

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        {
+            if (ModelState.IsValid)
+            {
+                try
+                {
+                    return await globalConfigurationMessagesService.CreateOrUpdateGlobalConfigurationMessages(globalConfigurationMessages);
+                }
+                catch (Exception ex)
+                {
+                    return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs:84]: need to change catch return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                }
+            }
+            return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.BadRequest, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.BadRequest.GetDescriptionAttribute(), Data = SystemMessages.msgBadRequest });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationMessagesController.cs:87]: need to change return response message

**Pull request closed without merging**